internal/resources/resource_host.go

```diff
@@ -47,6 +47,7 @@ const (
 	hPortalCommOkay       = "portal_comm_okay"
 	hPwrState             = "power_state"
 	hLabels               = "labels"
+	hSummaryStatus        = "summary_status"
```

Comment:
is there any specific reason for having 'summary' in label ?   'status' or 'instance_status' seems more suitable for me.

Reply:
summary_status is what the project_api (and therefore UI) returns, so this keeps them one-to-one.  summary_status is also what it is, since it is an aggregate result (in machine).

Reply:
In UI, it is 'Status.' IMO, if there is no plan to have multiple status fields - for single, aggregated, etc, 'summary' is unnecessary.  Otherwise LGTM. deferring to Mike for approval.
<img width="345" alt="image" src="https://user-images.githubusercontent.com/65302544/221975306-28f65a5c-0dd9-4dcd-90c1-7c1d1af772ed.png">

Reply:
the payload has summary_status
![host_summary_status](https://user-images.githubusercontent.com/86806285/221977602-901b19d9-e2b2-4094-8007-6fca77a7ce87.png)

Reply:
Backend & client API all references SummaryStatus, so this is fine as is.